### PR TITLE
New V1 Gateway Connector

### PR DESCRIPTION
**Pull request by davidkel**
Test only new V1 Gateway Connector

Signed-off-by: D Kelsey <d_kelsey@uk.ibm.com>

### CODE REVIEW

File: packages/caliper-fabric/test/connector-versions/v1/FabricGateway.js [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+                    invokerIdentity: 'User1',
+                    targetPeers: ['peer1', 'peer2']
+                };
+                await fabricGateway._sendSingleRequest(request);
+                Gateway.channel.should.equal('your-channel');
+                Transaction.submit.should.be.true;
+                Transaction.submitArgs.should.deep.equal(['arg1']);
+                Transaction.constructorArgs.should.equal('myFunction');
+                should.equal(Transaction.endorsingPeers, undefined);
+
```
Comment (nklincoln) [inline packages/caliper-fabric/test/connector-versions/v1/FabricGateway.js:386]: commented out to be reinstated later?

File: packages/caliper-fabric/lib/connector-versions/v1/FabricGateway.js [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+ *           are specified then targetPeers will take precedence
+ */
+
+/////////////////////////////
+// END OF TYPE DEFINITIONS //
+/////////////////////////////
+
+/**
+ */
+class V1FabricGateway extends ConnectorBase {
```
Comment (nklincoln) [inline packages/caliper-fabric/lib/connector-versions/v1/FabricGateway.js:71]: Do we want this to be named the same as the exported class?

**Review by nklincoln — CHANGES_REQUESTED** [review]
Only a few questions on this one 👍

File: packages/caliper-fabric/test/connector-versions/v1/FabricGateway.js (hunk 1 above)
Comment (davidkel) [reply packages/caliper-fabric/test/connector-versions/v1/FabricGateway.js:386]: Yep, thanks need to remove that

File: packages/caliper-fabric/lib/connector-versions/v1/FabricGateway.js (hunk 2 above)
Comment (davidkel) [reply packages/caliper-fabric/lib/connector-versions/v1/FabricGateway.js:71]: I was debating this with myself but would then see the need to split the connectors into 2 directories redundant (we would then also change the wallet names as well). I couldn't decide either so let it as is to see what comments came back :-)

**Review by nklincoln — APPROVED** [review]
🥇

**Pull request merged**